src/core/reactor.ts

```diff
@@ -149,11 +149,15 @@ export abstract class Reactor extends Component {
      * Return the location of the reactor instance in a bank, 
```

Comment:
Maybe we should update the comments as well?

Reply:
Yep. Will do!